hypertrace-graphql-labels-schema-api/src/main/java/org/hypertrace/graphql/label/schema/LabelApplicationRules.java

```diff
@@ -0,0 +1,19 @@
+package org.hypertrace.graphql.label.schema;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import org.hypertrace.core.graphql.common.schema.results.arguments.page.LimitArgument;
+import org.hypertrace.graphql.label.schema.rule.LabelApplicationRuleResultSet;
+
+@GraphQLName(LabelApplicationRules.TYPE_NAME)
+public interface LabelApplicationRules {
```

Comment:
Is this its own type? or only used indirectly

Reply:
Removed this class